index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Portfolio Mobile Version</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+       <main>
+            <!-- the toolbar or header section -->
+        <header>
+                <h4>My Logo</h4>
+               <!-- <img class="img" src="ham.png" alt="hambuger menu"> -->
+               <span class="hamb">&#9776;</span>
+        </header>
```

Comment:
You did a great job implementing the logo but kindly add a link to the logo redirecting to the home page! 💪